README.md

```diff
@@ -584,7 +584,7 @@ rate2: squants.market.CurrencyExchangeRate = USD/JPY 100.0
 
 scala> // OR
      | val rate3 = JPY(100) -> USD(1)
-rate3: squants.market.CurrencyExchangeRate = USD/JPY 100.0
+rate3: (squants.market.Money, squants.market.Money) = (100.0 JPY,1.0 USD)
```

Comment:
And, I shouldn't say "formats".  The actual type seems to be changing as well.  Perhaps some new behavior in Scala?